.golangci.toml

```diff
@@ -11,6 +11,9 @@ min-occurrences = 2
 [linters-settings.revive]
 ignore-generated-header = false
 
+[linters-settings.misspell]
+ignore-words = ["unknwon"]
```

Comment:
this was needed because the misspell linter was complaining about the `"github.com/unknwon/bra/cmd"` import